aws/etc/packer/tools/python/stardog/cluster/find_volume.py

```diff
@@ -55,7 +58,7 @@ def mount_format(device, mount_point):
             os.makedirs(home_dir, 0o775)
         except FileExistsError:
             pass
-        utils.command("chown -R ubuntu %s" % home_dir)
+        chown_rc, chown_err = utils.command("chown -R ubuntu %s" % home_dir)
```

Comment:
should we log or throw an exception if this fails?

Reply:
we can at least log it, I just updated this to work with the slight change to that method, but seems reasonable to make it known if this is hit